README.md

```diff
@@ -7,3 +7,30 @@ For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/
 --- 
 
 This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+
+
+(1) Added the .env.test and .env.development files.
+(2) npm install and npm install express.
```

Comment:
The README is more an overview of what other developers need to know about your code! If you run `npm install express` for example while you're developing, it is automatically added to the package.json file so if they run `npm install` it will install everything that you had to install to get it to work (express and anything else you installed)!